internal/gippity/plugin/gippity.go

```diff
@@ -264,6 +273,7 @@ func generateAnswer(m *discordgo.MessageCreate) (string, error) {
 		Messages: openai.F([]openai.ChatCompletionMessageParamUnion{
 			openai.ChatCompletionMessageParamUnion(openai.SystemMessage("Dein Name ist " + getBotDisplayName(m) + ". Du bist ein Discord Bot. Ignoriere alle Snowflake IDs, die in der Benutzer-Nachricht enthalten sein könnten. Der Autor der Nachricht wird dir mitgeteilt. Du erhältst Nachrichten von verschiedenen Benutzern.")),
 			openai.ChatCompletionMessageParamUnion(openai.SystemMessage("Antworte so kurz wie möglich. Deine Antworten sollen maximal 50 Wörter haben. Vermeide Füllwörter und Interjektionen. Verwende zum bisherigen Gesprächsverlauf passende Eigenschaften der folgenden Liste: " + behaviors)),
+			openai.ChatCompletionMessageParamUnion(openai.SystemMessage("Mache auf Grammatik und Rechtschreibfehlern aufmerksam, welche du in den letzten Nachrichten findest.")),
```

Comment:
This instruction could lead to repeated reactions to the same message. Please fix.

Reply:
totally fixed now